taurus_tangoarchiving/widget/tpgarchiving.py

```diff
@@ -56,8 +60,28 @@ def run():
 
     app = TaurusApplication(app_name='tpgArchiving')
     gui = TaurusGui()
+    gui.setWindowTitle('Taurus Archiving Plot')
     plot = TaurusPlot()
-
+    
+    #class MyPlot(TaurusPlot):
+        #pass
+        
+        ##def dropEvent(self, event):
+            ##print('In dropEvent(%s)' % str(dir(event)))
+        ##def dragMoveEvent(self,event):
+            ##print('Dragging ...')
+            ##event.acceptProposedAction()
+        ##def dragEnterEvent(self,event):
+            ##print('Entering ...')
+            ##event.acceptProposedAction()
+            
+    #plot = MyPlot()
+    #plot.setAcceptDrops(True)
+    #plot = fandango.qt.Dropable(TaurusPlot)()
+    #plot.setDropLogger(fandango.printf)
+    #plot.setSupportedMimeTypes(fandango.qt.MIMETYPES)
+    
+    plot.setBackgroundBrush(Qt.QColor('white'))
```

Comment:
IMHO, we should keep visual consistency among the various taurus* "oficial" projects (at least those under the taurus-org umbrella).

In that sense, I think that we should not be changing the background of the plots here, but intead do it either in taurus_pyqtgraph in general, or in a final application outside tango-archiving. 

If we want to do it in general, we need to consider things with some care (that is why it hasn't been done yet)... but I think this is a time as good as any other to start the discussion, so  I created [this issue in taurus_pyqtgraph](https://github.com/taurus-org/taurus_pyqtgraph/issues/56)

Reply:
the same comment is obviously valid for the other usages of `setBackgroundBrush()`